static/js/settings_notifications.js

```diff
@@ -2,17 +2,23 @@ var settings_notifications = (function () {
 
 var exports = {};
 
-var notification_settings = [
+var stream_notification_settings = [
+    {setting: "enable_stream_desktop_notifications", notifications:"desktop_notifications"},
+    {setting: "enable_stream_push_notifications", notifications:"desktop_notifications"},
```

Comment:
This should have been `push_notifications`.  Be very careful when refactoring -- this was a fairly serious bug that would likely not have been caught aside from reading the code. 

(This sort of bug is much more visible with smaller commits, which is why we require them in this project).